src/coreclr/src/vm/gcenv.ee.cpp

```diff
@@ -22,6 +29,43 @@ void GCToEEInterface::SuspendEE(SUSPEND_REASON reason)
     _ASSERTE(reason == SUSPEND_FOR_GC || reason == SUSPEND_FOR_GC_PREP);
 
     g_pDebugInterface->SuspendForGarbageCollectionStarted();
+    
+    if (GetIntConfigValue("GCGenAnalysisGen", nullptr, &gcGenAnalysisGen))
```

Comment:
Should this be cached? It does not sound right to be doing lookup by strings in the configuration hashtables during every suspend.